lib/services/services_linux.c

```diff
@@ -467,7 +467,6 @@ pipe_in_single_parameter(gpointer key, gpointer value, gpointer user_data)
 static void
 pipe_in_action_stdin_parameters(const svc_action_t *op)
 {
-    crm_debug("sending args");
```

Comment:
looks like a leftover from the debug-code I had added learning how this works when moving to usage of the service-library for fencing ;-)